### PR TITLE
First Sprint (October 29)

### PR DESCRIPTION
**Pull request by FrederikJorgensen**
First iteration of The Forgotten Giants project:

- Added routing
- Added screens
- Added responsiveness to components/screens
- AllGiants screen
- AboutGiant screen

### CODE REVIEW

File: screens/AllGiantsScreen/AllGiantsScreen.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+import * as Font from "expo-font";
+
+export default class AllGiantsScreen extends React.Component {
+  state = {
+    giants: [
+      {
+        id: 1,
+        name: "Teddy",
+        location: "HØJE TAASTRUP",
+        image: require("../../assets/images/Teddy.png")
```
Comment (mircealungu) [inline screens/AllGiantsScreen/AllGiantsScreen.js]: "../../assets/" happens 22 times in the project.
it is too much duplication. 
you could consider defining a function named asset() that concatenats "../../assets/" to a string..

File: screens/MapScreen/MapScreen.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        <MapView style={styles.mapStyle} />
+        <View style={styles.bottom}>
+          <Text>NEAREST TRAIN STATION: {this.props.location}</Text>
+        </View>
+      </View>
+    );
+  }
+}
+
+const styles = StyleSheet.create({
```
Comment (mircealungu) [inline screens/MapScreen/MapScreen.js]: Should be moved to MapScreenStyles?

File: screens/RewardScreen/RewardScreenStyles.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    marginTop: 30,
+    fontFamily: "amatic-sc",
+    fontSize: 40,
+    textAlign: "center"
+  },
+  img: {
+    width: "100%",
+    height: 363
+  },
+  buttonContainer: {
```
Comment (mircealungu) [inline screens/RewardScreen/RewardScreenStyles.js]: duplicated with RewardScrenStyles.js!

File: screens/RewardScreen/RewardScreen.js [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+        />
+        <View style={{ flexDirection: "row", marginTop: 15, marginLeft: 15 }}>
+          <Image
+            style={{ marginTop: 5 }}
+            source={require("../../assets/images/loc_emoji.png")}
+          />
+          <Text style={styles.text}>HØJE TAASTRUP</Text>
+        </View>
+
+        <Text style={styles.locationText}>
```
Comment (mircealungu) [inline screens/RewardScreen/RewardScreen.js]: Maybe refactor this in a few smaller subcomponents?
Also, if you'll want to make it prettier, it'll end up growing even more.

File: components/Giant/Giant.js [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+      "amatic-sc": require("../../assets/fonts/amatic-sc.ttf")
+    });
+    this.setState({ assetsLoaded: true });
+  }
+
+  render() {
+    return (
+      <View>
+        <View style={{ flexDirection: "row" }}>
+          <View>
```
Comment (mircealungu) [inline components/Giant/Giant.js]: Is there a point for this View ?

Comment (FrederikJorgensen) [reply components/Giant/Giant.js]: Yes. There must be only 1 Component in the render method. This is part of the design of React. That's why we wrap everything in 1 view.

**Pull request merged**